vtex-apps/my-account
# Fix/geolocation edit address

#### What did you change? \*

Updated some tooling by using `vtex setup` and changed the condition to clear the address' fields when geolocation is being used.

#### How to test it? \*

https://arthur--geraldinemolina.myvtex.com/account#/addresses

Use the workspace above. Then, try to add and edit an address. The address should be saved and edited normally.

#### Types of changes \*
